nikola/nikola.py

```diff
@@ -930,6 +929,23 @@ def init_plugins(self, commands_only=False, load_all=False):
                         bad_candidates.add(p)
                         utils.LOGGER.debug('Not loading compiler extension {}', p[-1].name)
             self.plugin_manager._candidates = list(set(self.plugin_manager._candidates) - bad_candidates)
+
+        # Find repeated plugins and discard the less local copy
+        def plugin_position_in_places(plugin):
+            for i, place in enumerate(self._plugin_places):
+                if plugin[0].startswith(place):
```

Comment:
could use some explainer comments that tell future readers what `plugin` is here, it took me a while to parse all this